cmd/nimbess-cni.go

```diff
@@ -142,6 +169,12 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
+	netns, err := ns.GetNS(args.Netns)
```

Comment:
this check doesnt need to be here as the agent will return a CNI error if the namespace doesnt exist